templates/kops-instancegroup-master.tpl.yaml

```diff
@@ -47,4 +47,19 @@ ${subnets}
       OnUnitActiveSec=60min
 
       [Install]
-      WantedBy=timers.target
\ No newline at end of file
+      WantedBy=timers.target
+  - name: calico-fix.service
+    manifest: |
+      [Unit]
+      Description=Calico network fix
+
+      [Service]
+      Type=oneshot
+      ExecStart=-/bin/bash -c "mv /etc/cni/net.d/10-calico.conflist /etc/cni/net.d/10-calico.conf"
+      ExecStartPost=-/bin/systemctl restart kubelet.service
+
+      [Path]
+      PathExists=/etc/cni/net.d/10-calico.conflist
```

Comment:
If I read the [docs](https://www.freedesktop.org/software/systemd/man/systemd.path.html), I expect this to be in a separate file called `calico-fix.path`, analogue to how the `timer` is set up.